src/drive/web/modules/drive/Container.jsx

```diff
@@ -102,11 +103,13 @@ const mapDispatchToProps = (dispatch, ownProps) => {
           __TARGET__ === 'mobile'
             ? {
                 action: files => dispatch(exportFilesNative(files)),
-                displayCondition: files =>
-                  files.reduce(
+                displayCondition: files => {
+                  if (isIOSApp()) return files.length === 1
```

Comment:
This will display it for a folder too right?

Reply:
Nice catch! Thanks. Fixed